src/dynamic_fmt.c

```diff
@@ -7463,7 +7463,7 @@ int dynamic_SETUP(DYNAMIC_Setup *Setup, struct fmt_main *pFmt)
 	pFmt->methods.salt_hash = salt_hash;
 	//pFmt->params.format_name = str_alloc_copy(Setup->szFORMAT_NAME);
 	pFmt->params.format_name = "";
-	pFmt->params.benchmark_length = 0;		// NOTE 0 'assumes' salted. If unsalted, we set back to -1
+	pFmt->params.benchmark_length = 7;		// If unsalted, we set to 0x107
```

Comment:
Maybe `BENCHMARK_LENGTH`.

Reply:
> Don't we need any changes to `--test-full` for it to process the new meaning of `benchmark_length` correctly?

I have no idea, I always hated that code for making a hopeless spaghetti mess of `formats.c`